frontend/src/actions/filters.js

```diff
@@ -0,0 +1,12 @@
+// FILTER_BY_RANGE
+export const setRangeFilter = (range = null) => ({
+    type: 'FILTER_BY_RANGE',
```

Comment:
A default range of `null` will result in range prop of `NaN` if passed through parseInt.